app/assets/stylesheets/base/_variables.scss

```diff
@@ -41,8 +41,9 @@ $light-blue: aliceblue;
 $light-blue-border: shade($light-blue, 5%);
 $dark-gray: #484343;
 $brown: #645858;
-$medium-gray: #aaa;
+$medium-gray: tint($brown, 20%);
 $light-gray: #ddd;
+$black-haze: #f7f7f7;
```

Comment:
Is this color name to avoid renaming the grays? I assumed it would be almost black until I saw the hex value.